package.json

```diff
@@ -14,6 +14,8 @@
     "lint": "lerna run lint",
     "publish": "node scripts/publish.js",
     "test": "lerna run test --concurrency 1",
+    "test:without-system-tests": "lerna run test --ignore system-tests --concurrency 1",
```

Comment:
@guw I did separate out the tests (system vs non-system vs all) so we can use this if we decide to control where the system tests run. Right now, I am using this to control that the system tests do not run on OS X on Travis.